### PR TITLE
cdrtools: Ensure kernel module loaded at boot

### PR DESCRIPTION
**Pull request by Vaelatern**
#4127 is it possible that the fix should be in the package itself?

### CODE REVIEW

**Comment by pullmoll** [comment]
Perhaps add an `INSTALL` to load the `sg` module? Otherwise a reboot is required.

**Comment by Vaelatern** [comment]
`        rm /usr/lib/modules-load.d/cdrtools.conf` Would this be necessary if the package owns the file?

**Comment by pullmoll** [comment]
Hmm.. no, it would not be necessary - my bad. Posting before the 1st coffee :)

**Comment by Vaelatern** [comment]
Added the INSTALL, not the remove. The `sg` module could legitimately be used by other programs, and is not provided in this package.

**Comment by xtraeme** [comment]
IMHO xbps should never attempt to load a kernel module via package scripts.

modules-load.d is enough. The user can run `modules-load` manually.

**Comment by pullmoll** [comment]
The master's voice :smile: Then perhaps a INSTALL.msg hint what's needed to make `cdrecord -scanbus` work without reboot would be useful?

**Comment by Vaelatern** [comment]
Is the solution to these build errors `nocross`?

**Comment by Vaelatern** [comment]
I'll add an `INSTALL.msg` informing the user they may wish to run `modules-load` to ensure all the necessary kernel modules are loaded.

File: srcpkgs/cdrtools/INSTALL [hunk 1]
```diff
@@ -0,0 +1,4 @@
+case "$ACTION" in
+	post)
+	    [ "$UPDATE" != "yes" ] && modprobe sg;;
+esac
```
Comment (xtraeme) [inline srcpkgs/cdrtools/INSTALL]: Please remove this INSTALL script.

**Comment by Vaelatern** [comment]
INSTALL script replaced with an INSTALL.msg
Should I set nocross on this package?

**Pull request closed without merging**